packages/cli/test/unit/commands/init.test.ts

```diff
@@ -0,0 +1,62 @@
+import init from '../../../src/commands/init';
+import { client } from '../../mocks/client';
+import { setupTmpDir } from '../../helpers/setup-unit-fixture';
+import { FetchOptions } from '../../../src/util/client';
+
+// The without setting this, the client.stderr output is undefined
+process.stdout.isTTY = true;
```

Comment:
The fact that I needed to add this has me feeling like I've done something wrong, I don't see the need for it in other tests